src/components/atoms/ButtonNG/ButtonNG.tsx

```diff
@@ -59,6 +63,27 @@ export const ButtonNG: React.FC<ButtonProps> = ({
   iconSize = "1x",
   title,
 }) => {
+  const history = useHistory();
+
+  const handleClick = useCallback(() => {
+    if (disabled) return;
+
+    if (linkTo) {
+      if (newTab) {
+        window.open(
+          linkTo,
+          externalUrlAdditionalProps.target,
+          externalUrlAdditionalProps.rel
+        );
+      } else history.push(linkTo);
```

Comment:
Not necessary to use `resolvedUrl`, since that's transformed from the originally supplied and may be an error in resolution